data.json

```diff
@@ -359,6 +368,15 @@
     "dateAdded": "2019-10-12T19:49:00.000Z",
     "tags": ["hacktoberfest", "accessories"]
   },
+  {
+    "name": "Python Discord",
```

Comment:
Please drop this opportunity, see https://github.com/swapagarwal/swag-for-dev/pull/388#issuecomment-538707977

Reply:
Done. I apologize for missing that as well.